windows/test-app.ts

```diff
@@ -0,0 +1,300 @@
+import * as path from 'path';
+import * as fs from 'fs';
+
+const windowsDir = 'windows';
+const reactNativeModulePath = findClosestPathTo(path.join('node_modules', 'react-native-windows'));
+const testAppNodeModulePath = findClosestPathTo(path.join('node_modules', 'react-native-test-app'));
+const srcRootPath = path.join(testAppNodeModulePath, windowsDir);
+const destPath = path.resolve('');
+
+
+function findClosestPathTo(fileOrDirName: string): string {
```

Comment:
Since you're returning `null`, the return type should be:
```suggestion
function findClosestPathTo(fileOrDirName: string): string | null {
```